### PR TITLE
564/dear poet

### PR DESCRIPTION
**Pull request by serundeputy**

### CODE REVIEW

**Comment by reynoldsalec** [comment]
@serundeputy can this weird sidebar layout of the sponsor images be fixed by editing the page?

Screen Shot 2019-04-18 at 9 56 23 AM

**Comment by serundeputy** [comment]
@reynoldsalec i don't think so; that is where `buildBasicPage` renders those fields; the user only has control of entering the images not the positioning

@dustinleblanc is that correct or am i missing something?

**Comment by reynoldsalec** [comment]
Ok, I think the thing that is confusing me is actually a style regression introduced here. The extra padding introduced to make the sidebar_image format work for the videos on /national-poetry-month/dear-poet is pushing the column images on other sidebar_image paragraphs down too much. See /national-poetry-month/poem-your-pocket-day for an example:

Screen Shot 2019-04-18 at 10 36 32 AM

File: components/SidebarTextImage.vue [hunk 1]
```diff
@@ -133,4 +135,8 @@ export default {
   margin-bottom: 15px;
   min-height: 116px;
 }
+/* video toc links */
+.sidebar-text-image .container .row .col-lg-4 {
```
Comment (reynoldsalec) [inline components/SidebarTextImage.vue]: This is probably our offending code for introducing the regression where sidebar_image right-hand column content is pushed down too much.

Comment (serundeputy) [reply components/SidebarTextImage.vue]: @reynoldsalec i've added a scoping class and pushed down toc that have the class `sidebar-top`.

QA: 
  * https://pr-399-sr7vzmq-tcynzice66pk2.us-2.platformsh.site/national-poetry-month/poem-your-pocket-day
  *

**Comment by pirog** [comment]
@serundeputy @reynoldsalec what is the status of this PR? Looks like we need to resolve some of thes e code related issues and fix a merge conflict. Let's make it happen!

**Pull request merged**